tests/clients/test_job.py

```diff
@@ -210,6 +271,64 @@ def test_job_summary_schema():
     assert data == JOB_SUMMARY
 
 
+def test_instance_size_schema():
+    data = InstanceSizeSchema().load(INSTANCE_SIZE_BODY)
+    assert data == INSTANCE_SIZE
+
+
+def test_job_parameter_schema():
+    data = JobParameterSchema().load(JOB_PARAMETER_BODY)
+    assert data == JOB_PARAMETER
+
+
+def test_job_command_schema():
+    data = JobCommandSchema().load(JOB_COMMAND_BODY)
+    assert data == JOB_COMMAND
+
+
+def test_job_definition_schema():
+    data = JobDefinitionSchema().load(JOB_DEFINITION_BODY)
+    assert data == JOB_DEFINITION
+
+
+@pytest.mark.parametrize(
+    "instance_size_type, instance_size",
+    [("m4.xlarge", INSTANCE_SIZE), ("custom", None)],
+)
+def test_job_definition_schema_invalid_instance_type(
+    instance_size_type, instance_size
+):
+    JOB_DEFINITION_BODY_INVALID = JOB_DEFINITION_BODY.copy()
```

Comment:
Since this is now being used as a local variable:
```suggestion
    job_definition_body_invalid = JOB_DEFINITION_BODY.copy()
```
or even just:
```suggestion
    invalid_body = JOB_DEFINITION_BODY.copy()
```